map/map.css

```diff
@@ -0,0 +1,24 @@
+body {
+    background-image: url("/assets/kathmandu-map.jpg");
+    background-size: cover;
+    background-position: center;
+
+}
+
+#flags {
+    display: flex;
+    justify-content: center;
+    margin-top: 400px;
+}
+
+img {
+    height: 300px;
+    width: 1000px;
+}
+#topflag {
+    display: flex;
+    justify-content: center;
+}
+
+
```

Comment:
In order to move around the links or completed quest spans, you need to target them in the css and give them position fixed or position relative or position absolute